1-js/11-async/04-promise-error-handling/article.md

```diff
@@ -135,71 +135,71 @@ new Promise((resolve, reject) => {
     alert("Can't handle such error");
 
 *!*
-    throw error; // throwing this or another error jumps to the next catch
+    throw error; // بعدی جهش می‌کند catch کردن این یا ارور دیگری به throw
 */!*
   }
 
 }).then(function() {
-  /* doesn't run here */
+  /* اینجا اجرا نمی‌شود */
 }).catch(error => { // (**)
 
   alert(`The unknown error has occurred: ${error}`);
-  // don't return anything => execution goes the normal way
+  // چیزی برنمی‌گرداند => اجرای برنامه به راه عادی خود ادامه می‌دهد
 
 });
 ```
 
-The execution jumps from the first `.catch` `(*)` to the next one `(**)` down the chain.
+اجرای برنامه از `.catch` اول `(*)` به بعدی `(**)` در انتهای زنجیره منتقل می‌شود.
 
-## Unhandled rejections
+## rejectionهای مدیریت نشده
```

Comment:
More attention here, pelase.

Reply:
I think if you don't translate it at all is better.

Reply:
> More attention here, pelase.
I think if you don't translate it at all is better.

```suggestion
## Unhandled rejections
```

Reply:
> More attention here, pelase.

I think if you don't translate it at all is better.

```suggestion
## Unhandled rejections
```